### ISSUE TITLE
adding after-render-complete emitter

### ISSUE THREAD

**Issue by Maikkiczk** [VadimDez/ng2-pdf-viewer#265]
- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [ ] Bug report -> please search issues before submitting
- [x] Feature request
- [ ] Documentation issue or request

Hi, I really need to catch the moment when the DOM is fully created. In fact, I looked in the code and found an interesting place to add the eventEmitter.
- The first when we have to make several pages
- The second when rendering a page.

**Proposal:**
For the rendering of a page, I put it in the renderPage function just before the return
```javascript
PdfViewerComponent.prototype.renderPage = function (pageNumber) {
      // ...
        this._pdf.getPage(pageNumber).then(function (page) {
           /* ... */
            pdfPageView.setPdfPage(page);
          _this.afterRenderComplete.emit(true); // --> here
          return pdfPageView.draw();
        });
    };
```

For rendering multiple pages, I had to put it in the updateSize function right after the pdfViewer._setScale function to be sure to have all the pages rendered.
```javascript
PdfViewerComponent.prototype.updateSize = function () {
        /* ... */
        this._pdf.getPage(this.pdfViewer.currentPageNumber).then(function (page) {
           /* ... */
            }
            _this.pdfViewer._setScale(scale, stickToPage);
            _this.afterRenderComplete.emit(true); // --> here
        });
    };
```

Like that, I can know when the DOM is created and i can manipulate it by catching the eventEmitter in the component.
```html
<pdf-viewer [src]="pdfSrc" [show-all]="true" (after-render-complete)="stateRender($event)"></pdf-viewer>
```

thanks in advance for your reply.

**Comment by VadimDez** [comment]
relates to #81

**Comment by VadimDez** [comment]
Released in `4.1.0`
https://github.com/VadimDez/ng2-pdf-viewer#page-rendered

**Comment by dinakarvedi** [comment]
Hi VadimDez, in my application i have mutliple page pdf's approximately upto 1000 pages how can i reduce the time to load the pdf's as fast as i can or else is there any chance of getting first ten pages in the inital page load and load rest of the pages
